content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+   secret share the same namespace, create the `istio-system` namespace with the
+   following command:
+
+    {{< text bash >}}
+    $ kubectl create namespace istio-system --context=${CTX_1}
+    {{< /text >}}
+
+1. Push the secret with the generated CA files to the cluster with the
+   following command:
+
+    {{< text bash >}}
+    $ kubectl create secret generic cacerts --context=${CTX_1} \
+        -n istio-system \
+        --from-file=${WORK_DIR}/${CLUSTER_1}/ca-cert.pem \
+        --from-file=${WORK_DIR}/${CLUSTER_1}/ca-key.pem \
+        --from-file=${WORK_DIR}/${CLUSTER_1}/root-cert.pem \
+        --from-file=${WORK_DIR}/${CLUSTER_1}/cert-chain.pem
+    {{< /text >}}
+
+Pushing the secret to the cluster overrides Istio's default CA, allowing you to
+establish trust between clusters using intermediate CAs that are signed by the
+shared root CA.
+
+**Congratulations!**
+
+You configured trust in the cluster to enable a multicluster deployment.
+
+### Deploy Istio
+
+Next, you must deploy Istio to the cluster if you hadn't already, or update your
+Istio configuration for multicluster. Istio requires the following configuration
+values to enable a multicluster deployment:
+
+<table>
+    <thead>
+        <tr>
+            <th>Configuration field</th>
+            <th>Description</th>
+            <th>Value</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>clusterName</code>
+            </td>
+            <td>Specifies a human-readable cluster name.</td>
+            <td><code>${CLUSTER_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>network</code></td>
+            <td>Specifies a network ID as an arbitrary string. All clusters in your mesh
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshID</code></td>
+            <td>Specifies a mesh ID as an arbitrary string. All clusters in your mesh share
+            the same mesh ID.</td>
+            <td><code>${MESH}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshExpansion</code></td>
+            <td>Recommended: Exposes the Istio control plane through the ingress gateway
+            of the mesh. Enable this option to connect remote clusters to a control
+            plane.</td>
+            <td><code>enabled: true</code></td>
+        </tr>
+    </tbody>
+</table>
+
+Using the [previously set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy Istio with the following steps:
+
+1. To pass configuration values to the Istio operator for installation,
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+    {{< text bash >}}
+    $ cat <<EOF> ${WORK_DIR}/${CLUSTER_1}/install.yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    spec:
+    values:
+        global:
+        meshID: ${MESH}
```

Comment:
everything below `global` falls under global requires tab for all lines after this

Reply:
I've addressed the indentation.